spec/pQueue.js

```diff
@@ -0,0 +1,84 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import pQueue from '../src/pQueue'
+
+chai.use(chaiChange)
+
+describe('pQueue', () => {
+  'use strict'
+
+  it('exists', () => {
+    expect(pQueue).to.be.a('function')
+  })
+
+  context('enqueue()', () => {
+    it('adds an element with priority to the back of the queue.', () => {
+      const aQueue = new pQueue()
+
+      aQueue.enqueue('dude', 90)
+      aQueue.enqueue('cool', 20)
+      aQueue.enqueue('turd', 50)
+      expect(aQueue.enqueue('foo',70)).to.eql([{name: 'dude', priority: 90}, {name: 'cool',  priority: 20}, {name: 'turd', priority: 50}, {name: 'foo', priority: 70}])
+    })
+  })
+
+  // context('front()', () => {
```

Comment:
is this a WIP? are you going to remove this commented code or use it in the future?